test/unit/CMakeLists.txt

```diff
@@ -160,9 +160,14 @@ add_executable(test_network
   main.cpp
   test_network.cpp)
 if(CURL_ENABLED)
-  include_directories(${CURL_INCLUDE_DIR})
   target_compile_definitions(test_network PRIVATE -DCURL_ENABLED)
-  target_link_libraries(test_network PRIVATE ${CURL_LIBRARY})
+  if( CMAKE_VERSION VERSION_LESS 3.11 AND CMAKE_CROSSCOMPILING )
+      # Hack needed for ubuntu:18.04 mingw64 cross compiling to avoid
+      # -isystem to be emitted (similar to https://discourse.cmake.org/t/use-of-isystem/1574)
+      target_link_libraries(test_network PRIVATE ${CURL_LIBRARIES})
+  else()
+      target_link_libraries(test_network PRIVATE CURL::libcurl)
+  endif()
```

Comment:
Does this test use curl on its own? If yes, we should continue to provide `CURL_INCLUDE_DIRS` along `CURL_LIBRARIES`. If no, we don't need to be explicit about curl's link libraries.

Reply:
> Does this test use curl on its own

yes it does, and also uses CURL headers. fixed to add CURL_INCLUDE_DIRS